src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -273,6 +274,31 @@ export default class ClinicalData extends React.Component<
         this.horizontalBars = !this.horizontalBars;
     }
 
+    @action.bound
+    private onClickShowNA() {
+        this.showNA = !this.showNA;
+    }
+
+    private readonly clinicalDataPromiseFiltered = remoteData({
+        await: () => [
+            this.clinicalDataPromise,
+            this.props.store.samples
+        ],
+        invoke: async () => {
```

Comment:
you can take the async off here since you are not doing anyting async inside.   The affect of putting async here is actually that there's a promise wrapped inside of a promise.  it doesn't matter but ...
note, you will still need to return the promise below.